docs/contrib/DeveloperGuide.md

```diff
@@ -0,0 +1,7 @@
+# Developer Guide
```

Comment:
I think this file can be removed on both docs.

Reply:
I added logging to this file and kept it as root for the developer's guide section as I didn't know where to add the logging info and I need a file as an entry for the developer guide section. Please let me know your thoughts